templates/index.html

```diff
@@ -36,21 +17,22 @@
 
     <div class="meetup">
 
-      <h3 class='meetup-name'><a href="{{ meetup.meetup_link }}">{{ meetup.name }}</a></h3>
+      <h3 class='meetup-name'><a href="{{ meetup.get_meetup_link }}">{{ meetup.get_meetup_name }}</a></h3>
 
-      {% if meetup.next_event_topic %}
+      {% if meetup.get_meetup_next_event_topic %}
         <div class='meetup-description'>
           <strong class='subheader'>Next Event:</strong>
-          <p>{{ meetup.next_event_topic }} - {{ meetup.datetime}} at
-          <a href="{{ meetup.map_link }}">{{ meetup.venue }}</a></p>
+          <p>{{ meetup.get_meetup_next_event_topic }} - {{ meetup.get_meetup_datetime}} at
+          <a href="{{ meetup.get_meetup_map_link }}">{{ meetup.get_meetup_venue }}</a></p>
         </div>
       {% endif %}
 
       <div class='meetup-pizza-places'>
         <strong class='subheader'>Pizzas:</strong>
-        {% for place in meetup.pizza_places %}
+        {% for pizza_place in meetup.get_meetup_pizza_places %}
         <li class='meetup-pizza-list'>
```

Comment:
Look for an HTML5 validator, and consider running your generated HTML through one as a regular part of your development process.